src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.Tests/inputs/NugetConfigFiles/EnsureAppendsDisableEntryAfterLastClear/NuGet.input.config

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This file should be kept in sync across https://www.github.com/dotnet/wpf and dotnet-wpf-int repos. -->
+<configuration>
+  <packageSources>
+    <clear />
+    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <add key="darc-int-dotnet-arcade-b043797" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-arcade-b0437974/nuget/v3/index.json" />
```

Comment:
You're passing this feed as a parameter to the test, so the test should add it to the nuget.config, I think it should be removed from here.

Reply:
Resolving this one as https://github.com/dotnet/arcade-services/pull/1334/files#r467226321 has a better explanation of what I mean